addons/godot-xr-tools/functions/Function_pointer.gd

```diff
@@ -142,6 +143,7 @@ func _process(delta):
 	
 	if enabled and $RayCast.is_colliding():
 		var new_at = $RayCast.get_collision_point()
+		var top_bottom = get_parent().get_joystick_axis(1)
 		
```

Comment:
this should be set with an export property, not hardcoded to 1

Reply:
oh, right my solution also won't support horizontal scrolling if we don't do that. I had it hardcoded since I knew I wanted it to be up-down on the joystick. left-right was for rotating the camera